m3.yml

```diff
@@ -74,52 +74,52 @@ classes:
 
 properties:
   - creator:
-    display_label: "Creator(s)"
-    definition: "A person or organization responsible for creating the resource. (This is an example definition for a creator metadata field.)"
-    usage_guidelines: "Record in lastname, firstname order."
-    requirement: "recommended, if applicable"
-    controlled_value:
-      format: "xsd:literal"
-      sources:
-        - "/qa/terms/local/roles/"
-    sample_value: Smith, John
-    property_uri: http://purl.org/dc/elements/1.1/creator
-    available_on:
-      class:
-        - Collection
-        - MyCustomWorkType
-    range:
-      class_uri: http://id.loc.gov/ontologies/bibframe/Agent
-    data_type: xsd:string
-    syntax: edtf
-    cardinality:
-      minimum: 0
-      maximum: 100
-    index_documentation: "searchable, displayable, creator facet"
-    validations:
-      match_regex: "[\\d]{4}"
-    mapping:
-      dpla: "http://purl.org/dc/elements/1.1/creator"
+      display_label: "Creator(s)"
```

Comment:
I forgot that
```yaml
- key:
   key2: value
```
is parsed differently than 
```yaml
- key:
    key2: value
```
the former is an array with two key-value pairs: `[ key: '', key2: 'value' ]`, while the latter is a key-value pair: `key: { key2: 'value' }`